samples: blink_led/fade_led: Update when no on-board PWM LED (#22277)
zephyrproject-rtos/zephyr: samples/basic/blinky_pwm/README.rst

   Unsupported board: pwm_led0 devicetree alias is not defined

> In this case, you can still make this application working on your board by

Reviewer: ```suggestion
In this case, you can still make this application work on your board by
```

> wiring one manually to an PWM enabled pin and use a devicetree overlay to

Reviewer: ```suggestion
wiring an LED manually to a PWM enabled pin and using a devicetree overlay to
```

reflect this configuration.

> An example of .overlay file is available in current folder and works with

Reviewer: ```suggestion
An example .overlay file is available in the boards/ folder and works with
```

> An example of .overlay file is available in the /boards folder and works with

Reviewer 2: I guess it was supposed to be "boards/".

Author: Fixed, txs!

After flashing, the sample starts blinking the LED as described above. It also
prints information to the board's console.

Alternative configuration for board without on-board LED connected to PWM
*************************************************************************

> If there is no on-board LED connected to a PWM configured PIN on your board,

Reviewer: Maybe just put this where the table was?

> you can wire one manually and use a .overlay file to configure LED-PWM driver.

Reviewer: ```suggestion
you can wire one manually and use a devicetree overlay to configure this application.
```

An example of .overlay file is available in current folder and works with

> nucleo_f401re.

Reviewer: ```suggestion
nucleo_f401re. See :ref:`set-devicetree-overlays` for more information.
```
